tests/test.py

```diff
@@ -128,6 +128,15 @@ def fetch_json(self, path, params={}, **kwargs):
 # TODO: split the handler tests into another file
 
 
+class TestRootHandler(BaseTestCase):
+    def test_home_page(self):
+        response = self.fetch('/')
+        files = open("index.html", "rb")
```

Comment:
Why is this plural? Also, simply use something like `expected_output = open('index.html').read()`.

Reply:
I could not use File as travis did not allow me to and I was afraid that file is a keyword so I used files. Okay I can change it.

Reply:
Then something like `index_file` is preferable.

Reply:
I am using f now. Is it okay?

Reply:
As I said in my first comment, you don't even need an intermediate file.

Reply:
okay but how do I close the opened file then?

Reply:
I gives an error that file is not closed

Reply:
It gets automatically closed.

Reply:
`expected_output = open('index.html').read()` results in an error? Can you provide the exact error output.

Reply:
Sorry my bad its a warning. It says :
```
ResourceWarning: unclosed file <_io.TextIOWrapper name='index.html' mode='r' encoding='UTF-8'>
  expected_output = open('index.html').read()
```

Reply:
Should I go with this?

Reply:
Huh, that's strange. It should just be closed on refcount==0. In any case, use `with`. In Python3, there's not much of a reason to manually call `close()` on a file.